src/Controls/src/Core/TypedBinding.cs

```diff
@@ -145,7 +149,11 @@ internal override BindingBase Clone()
 			if (handlers != null)
 			{
 				for (var i = 0; i < _handlers.Length; i++)
+				{
+					if (_handlers[i] == null)
```

Comment:
Should this one be `handlers`:
```suggestion
					if (handlers[i] == null)
```

Reply:
about the same